lib/rest/RESTClient.js

```diff
@@ -0,0 +1,3506 @@
+"use strict";
+
+const ApplicationCommand = require("../structures/ApplicationCommand");
+const Channel = require("../structures/Channel");
+const Endpoints = require("./Endpoints");
+const ExtendedUser = require("../structures/ExtendedUser");
+const Guild = require("../structures/Guild");
+const GuildAuditLogEntry = require("../structures/GuildAuditLogEntry");
+const GuildIntegration = require("../structures/GuildIntegration");
+const GuildPreview = require("../structures/GuildPreview");
+const GuildTemplate = require("../structures/GuildTemplate");
+const GuildScheduledEvent = require("../structures/GuildScheduledEvent");
+const Invite = require("../structures/Invite");
+const Member = require("../structures/Member");
+const Message = require("../structures/Message");
+const Permission = require("../structures/Permission");
+const PrivateChannel = require("../structures/PrivateChannel");
+const RequestHandler = require("./RequestHandler");
+const Role = require("../structures/Role");
+const StageInstance = require("../structures/StageInstance");
+const ThreadMember = require("../structures/ThreadMember");
+const User = require("../structures/User");
+const AutoModerationRule = require("../structures/AutoModerationRule");
+const {formatAllowedMentions, processAttachments} = require("../util/util");
+const {setTimeout: sleep} = require("node:timers/promises");
```

Comment:
dtto (universal JS suits better here)
```suggestion
const sleep = (ms) => new Promise((res) => setTimeout(res, ms));
```